src/labels/CustomLabels.labels

```diff
@@ -1457,6 +1457,14 @@
         <shortDescription>NameRequired</shortDescription>
         <value>Subject is a required field.</value>
     </labels>
+    <labels>
```

Comment:
![Screen Shot 2021-02-23 at 7 40 31 PM](https://user-images.githubusercontent.com/57271120/108944149-155dfc00-760f-11eb-8bf4-f943519e6bcc.png)
![Screen Shot 2021-02-23 at 7 40 41 PM](https://user-images.githubusercontent.com/57271120/108944152-15f69280-760f-11eb-93ae-87d4020f5bdf.png)
@mattwashington @sam-knox Can you review the new custom labels' descriptions

Reply:
Looks good to me! Good work.
@sam-knox Do you have a moment to verify?

Reply:
Yep, these labels look good to me too!